ref/general/requestTiming.adoc

```diff
@@ -0,0 +1,71 @@
+:page-layout: general-reference
+:page-type: general
+:page-description: The request timing feature automatically detects slow requests and logs the information when the request is completed.
+:page-categories: MicroProfile
+:seo-title: The request timing feature automatically detects slow and hung requests
+:seo-description: The request timing feature automatically detects slow and hung requests and logs the information when the request is completed.
+= Slow and hung request detection
+
+Slow or hung requests can happen, and your application needs a continuous monitoring tool to detect the cause.
+The request timing feature automatically detects slow  and hung requests and logs the information when the request is completed.
+
+The `requestTiming` feature monitors requests with respect to time and automatically detects and diagnoses slow and hung requests.
+In addition to the previous capability of detecting the slow request, the updated feature now also logs when the request is completed, thus improving the monitoring efficiency.
+The users can view the information about a completed slow request with the request id and time of completion.
+Similarly, in the case of a hung request, a message is logged if a request hung initially, gets completed.
+
+== Configure slow and hung request detection
+
+You can enable the request timing the feature by adding the `requestTiming` to the list of features in the `server.xml` file.
+
+[source,java]
+----
+<featureManager>
+      ...
+      <feature>requestTiming-1.0</feature>
+      ...
+</featureManager>
+----
+
+The slow and hung request thresholds are configured by setting `slowRequestThreshold` and `hungRequestThreshold` in the `<requestTiming/>` configuration in the `server.xml` file.
+This setting sets a single slow and hung request threshold for all requests that are timed.
```

Comment:
not sure single is correct because each attribute sets a threshold- so there should be two?

These attributes specify global slow and hung request thresholds for all requests that are timed.